<ui version="4.0" >
 <class>Config</class>
 <widget class="QDialog" name="Config" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configuration</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Programme d'encodage:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Options:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLineEdit" name="command" />
         </item>
         <item>
          <widget class="QLineEdit" name="commandOptions" />
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Dans les options, $STREAM représente le flux et $OUT le fichier de sortie.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Répertoire des enregistrements:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLineEdit" name="repertoire" />
     </item>
     <item row="0" column="3" >
      <widget class="QToolButton" name="choixRepertoire" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Fichier XML des programmes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="nomFichierXML" />
     </item>
     <item row="1" column="3" >
      <widget class="QToolButton" name="choixFichierXML" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Format du nom fichier par défaut:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2" >
      <widget class="QLineEdit" name="nomFichier" />
     </item>
     <item row="1" column="4" >
      <widget class="QToolButton" name="populateDB" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="../ressources/ressources.qrc" >:/images/images/refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%n&lt;/span> : Nom de la chaine  &lt;span style=" font-weight:600;">%t&lt;/span> : Titre du programme  &lt;span style=" font-weight:600;">%a&lt;/span> : Année &lt;span style=" font-weight:600;">%m&lt;/span> : Mois  &lt;span style=" font-weight:600;">%j&lt;/span> : Jour&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>Fichier XML disponible ici :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>&lt;a href="http://xmltv.myftp.org/download/tnt.zip">http://xmltv.myftp.org/download/tnt.zip&lt;/a>
</string>
       </property>
       <property name="openExternalLinks" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="demarrerEnIcone" >
       <property name="text" >
        <string>Démarrer iconifié</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>515</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok" >
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="annuler" >
       <property name="text" >
        <string>&amp;Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources/ressources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>Config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>344</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel" >
     <x>238</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>annuler</sender>
   <signal>clicked()</signal>
   <receiver>Config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>429</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel" >
     <x>238</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
